collector/systemd_linux.go

```diff
@@ -55,6 +56,10 @@ func NewSystemdCollector() (Collector, error) {
 		prometheus.BuildFQName(namespace, subsystem, "unit_state"),
 		"Systemd unit", []string{"name", "state"}, nil,
 	)
+	unitStartedAtDesc := prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, subsystem, "unit_started_at_seconds"),
```

Comment:
Let's call this unit_start_time_seconds to be consistent with already existing `process_start_time_seconds`.

Reply:
@discordianfish That won't work, because `process_start_time_seconds` metric family has no labels, so the collector will fail.

Reply:
I think for consistency, we could name it `unit_start_time_seconds`.

Reply:
@SuperQ That's what I suggested! Should have put it in ```` though :)

Reply:
Sorry! I didn't read what you said closely enough. 🦆